pkg/migrate/main.go

```diff
@@ -0,0 +1,320 @@
+package migrate
```

Comment:
we have another thing we call "migrate" (for db migrations)
so, try to think of a better name .

also, how compose-oriented is this? 
we are dealing 99% with records and 1% with users, correct?
can it handle other types of data?
can it handle other formats (yaml)?

Reply:
At the moment it's as you said; mostly Compose oriented, but.. It should be fairly trivial to extend this to an arbitrary structure and arbitrary data source